environments/prod/platform-hmcts-net.yml

```diff
@@ -812,15 +809,15 @@ cname:
   - name: "_ea7ddc228444799015f19f0b8187f919.www.paybubble"
     ttl: 300
     record: "918AE6DB4F60E56D6E92DB2F5F39E3AA.93CEB9C2DCDAAB64E151F2A7A1851C97.dfa7868c9f76acaf214a.comodoca.com."
-  - name: "_18bf7504025b44083d0aa4efbf13ff78.www.moneyclaim-legal"
+  - name: "_6b5a4b25400cfd39eeb8497f6f1b65ae.www.moneyclaim-legal"
     ttl: 300
-    record: "5B8098406E066B1E5F42D5CA3D63D949.B37E974F8BDF9C371596B20E0ACFE5A4.de9cf7fc237bf2217c1c.comodoca.com."
-  - name: "_18bf7504025b44083d0aa4efbf13ff78.moneyclaim-legal"
+    record: "3D8E7307FB268526F6123C1940D117A3.5339B0A08C67EDAFB0A94A60DF873F73.0ceed3b289e76c7f6ac4.comodoca.com."
+  - name: "_6b5a4b25400cfd39eeb8497f6f1b65ae.moneyclaim-legal"
     ttl: 300
-    record: "5B8098406E066B1E5F42D5CA3D63D949.B37E974F8BDF9C371596B20E0ACFE5A4.de9cf7fc237bf2217c1c.comodoca.com."
-  - name: "_18bf7504025b44083d0aa4efbf13ff78.tmwww.moneyclaim-legal"
+    record: "3D8E7307FB268526F6123C1940D117A3.5339B0A08C67EDAFB0A94A60DF873F73.0ceed3b289e76c7f6ac4.comodoca.com."
+  - name: "_6b5a4b25400cfd39eeb8497f6f1b65ae.tmwww.moneyclaim-legal"
```

Comment:
this isn't in use

Reply:
2 days left till the cert expires, better to just get it with the extra SAN